src/components/inputs/BasicInput/BasicInput.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../../styles/_partials/index';
+
+.BasicInput {
+    position: relative;
+    display: block;
+    margin-bottom: 15px;
+
+    input {
+        @include flywheelInput;
+    }
+}
```

Comment:
Not sure if our editor rules are being ignored by editors, but would you change these indentations from spaces to tabs?